tests/app/user/test_rest.py

```diff
@@ -791,30 +791,20 @@ def test_send_user_confirm_new_email_returns_400_when_email_missing(client, samp
     mocked.assert_not_called()
 
 
-@pytest.mark.parametrize('data,email_access_validated_at', [
-    ({'_password': '1234567890'}, datetime(2020, 2, 13, 12, 0)),
-    ({
-        '_password': '1234567890',
-        'validated_email_access': True,
-    }, datetime(2020, 2, 14, 12, 0)),
-    ({
-        '_password': '1234567890',
-        'validated_email_access': False,
-    }, datetime(2020, 2, 13, 12, 0))
-])
 @freeze_time('2020-02-14T12:00:00')
-def test_update_user_password_saves_correctly(client, sample_service, data, email_access_validated_at):
+def test_update_user_password_saves_correctly(client, sample_service):
     sample_user = sample_service.users[0]
     sample_user.email_access_validated_at = datetime(2020, 2, 13, 12, 0)
```

Comment:
Do we need to bother about this attribute at all in these tests?

Reply:
Nope – squashed out